demonstrations/tutorial_fdhs.py

```diff
@@ -0,0 +1,455 @@
+r"""Fixed Depth Hamiltonian Simulation via Cartan Decomposition
+===============================================================
+
+We introduce the powerful Lie theoretic decomposition technique for Hamiltonians, :math:`H = K h K^\dagger`,
+that lets you time-evolve by arbitrary times with fixed depth, :math:`e^{-i t H} = K e^{-i t h} K^\dagger`.
+In particular, we follow the approach in [#Kökcü]_ that directly provides us with a (fixed depth) circuit
+decomposition of the unitaries :math:`K` and :math:`e^{-i t h}`.
+
+Sounds too good to be true? There are of course caveats, mostly of practical nature.
+One of them is that the Lie algebra of H, in terms of which
```

Comment:
```suggestion
One of them is that the Lie algebra of :math:`H`, in terms of which
```

In addition, the "in terms of which this decomposition is based" reads a bit funny to me personally.
My gut feeling was "in terms on which.." but that also doesn't sound right.

Reply:
I just removed the Hamiltonian bit completely and simplified the sentence